source/connect.txt

```diff
@@ -38,133 +37,118 @@ This is equivalent to the following command:
+For example, run the following command to authenticate on the ``admin``
+database as user ``alice``:
 
-.. note::
+.. code-block:: sh
 
-   To provide a password with the command instead of using the masked 
-   prompt, you can use the ``--password`` option.
+   mongosh "mongodb://mongodb0.example.com:28015" --username alice --authenticationDatabase admin
 
 .. seealso::
 
-   - :manual:`Enable Access Control </tutorial/enable-authentication/>` 
-     to enforce authentication.
-   - Add :manual:`Database Users </core/security-users/>` 
-     to provide authenticated access to a MongoDB instance.
+   - To enforce authentication on a deployment, see
+     :manual:`Enable Access Control </tutorial/enable-authentication/>`.
+
+   - To provision access to a MongoDB deployment, see :manual:`Database
+     Users </core/security-users/>`.
 
 Connect to a Replica Set
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-To connect to a replica set:
+To connect to a replica set, perform one of the following actions:
 
-- If you are using the :manual:`DNS Seedlist Connection Format 
-  </reference/connection-string/#dns-seedlist-connection-format>`, you 
-  can include the ``+srv`` modifier in your connection string.
+- If you use the :manual:`DNS Seedlist Connection Format
+  </reference/connection-string/#dns-seedlist-connection-format>`,include
+  the ``+srv`` modifier in your connection string.
 
-  .. example::
+  For example, run the following command to connect to a replica set on
+  ``server.example.com``:
+
+  .. code-block:: sh
 
-     .. code-block:: sh
+     mongosh "mongodb+srv://server.example.com/"
 
-        mongosh "mongodb+srv://server.example.com/"
+  .. note:: +srv TLS Behavior
 
-  .. note::
-     Using the ``+srv`` connection string modifier automatically sets 
-     the 
-     :manual:`tls option </reference/connection-string/#urioption.tls>` 
-     to ``true`` for the connection. You can 
-     override this behavior by explicitly setting ``tls`` to ``false``.
+     When you use the ``+srv`` connection string modifier, MongoDB
+     automatically sets the :option:`--tls <--tls>` connection option to
+     ``true``. To override this behavior, set ``--tls`` to ``false``.
 
 - You can specify the replica set name and members in the
   :manual:`connection string </reference/connection-string>`.
 
-  .. example::
+  For example, run the following command to connect to a three-member
+  replica set named ``replA``:
 
-     To connect to a three-member replica set named ``replA``:
-
-     .. code-block:: sh
+  .. code-block:: sh
 
-        mongosh "mongodb://mongodb0.example.com.local:27017,mongodb1.example.com.local:27017,mongodb2.example.com.local:27017/?replicaSet=replA"
+     mongosh "mongodb://mongodb0.example.com.local:27017,mongodb1.example.com.local:27017,mongodb2.example.com.local:27017/?replicaSet=replA"
 
-.. note::
+.. note:: directConnection Parameter
```

Comment:
[questions] Does this note apply to only the second option above, or to both? It's not clear until you read the whole note. Also, should we title this note a little more specifically? Perhaps, "directConnection Parameter added automatically" or something like that?